content/en/blog/falco-aws-security-lake/index.md

```diff
@@ -0,0 +1,177 @@
+    securitylake:
+      bucket: "arn:aws:s3:::aws-security-data-lake-{region}-{id}" # Bucket for AWS SecurityLake data, if not empty, AWS SecurityLake output is enabled
+      region: "{region}" # Region
+      prefix: "falco" # Prefix for keys
+      accountid: "{accountId}" # Account ID
+      interval: 5 # Time in minutes between two puts to S3 (must be between 5 and 60min)
+      batchsize: 1000 # Max number of events by parquet file
+      minimumpriority: "" # minimum priority of event to use this output, order is `emergency|alert|critical|error|warning|notice|informational|debug or ""`
+```
+
+Few details about the settings:
+- `podAnnotations`: If you use an OIDC provider for your EKS cluster, you can associate the serviceAccount of Falcosidekick with an IAM Role, replace {accountId} and {role} by your relevant values
+- `accesskeyid`, `secretaccesskey`: must be set if you use keys to access to AWS API, might be useful outside the AWS Ecosystem, we encourage you to use an instance profile or IAM role instead
+- `rolearn`: set it to specify the role Falcosidekick will use, useless if you use an instance profile or IAM role
+- `region`: specify the region to use to create the session, useless if you use an instance profile or IAM role
+- `checkidentity`: Falcosidekick will try its accesses to AWS at bootstrap, if it fails, all AWS outputs are disabled
+- `securitylake.bucket`: the bucket name returned by AWS Security Lake at Custom source creation (mandatory to enable the output)
+- `securitylake.region`: the region of the events, as a single bucket can store events from different regions (mandatory)
+- `securitylake.accountid`: the accountId of the account where the events have been created, as a single bucket can store events from different accounts (mandatory)
+- `securitylake.interval`: time in minutes between 2 PUTs of parquet files into S3 to follow the service's requirements
+- `securitylake.batchsize`: max number of events by parquet file, to avoid to reach max allowed size
+- `securitylake.minimumpriority`: filter the events we send to AWS Security Lake by their priority
+
+## Deployment of Falco & Falcosidekick
+
+These classic Helm commands will deploy everything for you:
+
+```shell
+helm repo add falcosecurity https://falcosecurity.github.io/charts
+helm repo update
+helm install falco -n falco -f values.yaml --create-namespace
+```
+
+Falco and Falcosidekick will be installed and you should get in the Falcosidekick pods the following logs:
+
+```
+2023/03/13 13:03:06 [INFO]  : Enabled Outputs : [AWSSecurityLake]
+2023/03/13 13:03:06 [INFO]  : Falco Sidekick is up and listening on :2801
+```
+
+Falcosidekick is ready to receive the events and put them to AWS Security Lake!
+
+## Results
+
+### Logs of Falcosidekick
+
+When Falco will trigger some of its rules, Falcosidekick will receive and queue them for 5 minutes, then it will create a parquet file and put into the S3 bucket:
```

Comment:
```suggestion
When Falco triggers any rules, Falcosidekick will receive and queue them every 5 minutes, then it will create a parquet file and load them into the S3 bucket:
```